lib/user_data_parser/output_handler/output_handler.rb

```diff
@@ -0,0 +1,8 @@
+module UserDataParser
+  class OutputHandler
+    def print(string)
+      puts string
```

Comment:
Is output handler just I/O? I'm glad you're thinking about dependency inversion, but this seems like it might be a premature abstraction. Moving forward, I think it's a good practice to wait until we see signs in the code that call for abstraction. It looks like this is only called in one file at the moment, so it's probably unnecessary. Let's leave it in though, but can you rename to something that calls out what it's abstracting a little more explicitly? Maybe even just ```IO```?